src/main/java/frc/robot/subsystems/arm/commands/SetArmsPosition.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.arm.ArmConstants;
+import frc.robot.subsystems.arm.ArmKinematics;
+import frc.robot.subsystems.arm.ArmSystemModel;
+import frc.robot.subsystems.arm.PrototypeArm;
+
+public class SetArmsPosition extends CommandBase {
+    private final PrototypeArm prototypeArm;
+    private final Translation2d position;
+
+    public SetArmsPosition(PrototypeArm prototypeArm, Translation2d position) {
+        this.prototypeArm = prototypeArm;
+        this.position = position;
+        addRequirements();
+    }
+
+    @Override
+    public void execute() {
+        prototypeArm.setPosition(position);
+    }
+}
```

Comment:
Where is the end?

Reply:
IIUC you _don't_ want to set the powers to zero, because then the motor won't actively hold the subsystem in place.
On the other end, `isFinished()` does seem like it's missing here.

Reply:
I'm pretty sure that if you set the motors to break the arm won't fall

Reply:
It's possible that's the case in the prototype because the arms are so light, but I don't imagine this will be the case in the real arm.
Anyway, I'd not set the power to zero, it doesn't "cost" you anything (and the opposite does "cost" you an additional line of code).